### ISSUE TITLE
Xcode error on build: "15 duplicate symbols for architecture x86_64"

### ISSUE THREAD

**Issue by simensol** [getsentry/sentry-cocoa#421]
_Platform:_
- [X]  iOS
- [ ]  tvOS
- [ ]  MacOS
- [ ]  watchOS

_Swift:_
- [X]  Yes -> If yes, which version? 5.2
- [ ]  No

_sentry-cocoa installed with:_
- [ ] CocoaPods
- [ ] Carthage
- [ ] Manually
- [X] SwiftPM

Version of sentry-cocoa:
**5.0.0-beta.0**

---
**I have following issue:**

After installing Sentry in Xcode using SwiftPM, I get an error when I try to build the iOS app.

**Steps to reproduce:**
- Install Xcode 11.4
- Create a Tabbed App iOS project
- Install Sentry 5.0.0-beta.0 using SwiftPM
- Build app

**Actual result:**
- Xcode throws the following error:

```
duplicate symbol '_OBJC_IVAR_$_SentryEnvelope._items' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_IVAR_$_SentryEnvelopeItem._header' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_IVAR_$_SentryEnvelope._header' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_CLASS_$_SentryEnvelopeItemHeader' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_METACLASS_$_SentryEnvelopeItemHeader' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_CLASS_$_SentryEnvelopeHeader' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_METACLASS_$_SentryEnvelopeHeader' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_CLASS_$_SentryEnvelopeItem' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_METACLASS_$_SentryEnvelopeItem' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_IVAR_$_SentryEnvelopeItemHeader._length' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_IVAR_$_SentryEnvelopeItemHeader._type' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_CLASS_$_SentryEnvelope' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_METACLASS_$_SentryEnvelope' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_IVAR_$_SentryEnvelopeHeader._eventId' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
duplicate symbol '_OBJC_IVAR_$_SentryEnvelopeItem._data' in:
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryEnvelope.o
    /Users/Username/Library/Developer/Xcode/DerivedData/Procraster-crtefmaxuhmlnsduqghfblkekdxn/Build/Intermediates.noindex/Sentry.build/Debug-iphonesimulator/Sentry.build/Objects-normal/x86_64/SentryTransport.o
ld: 15 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Expected result:**
Successful build.

**Comment by stella-mylermedia** [comment]
I have the same issue. Installed Sentry using cocoapods though.

**Comment by HazAT** [comment]
I just tried it on a clean new iOS project and can't reproduce it.
Can you make sure you cleaned your cache completely with `Cmd+Shift+Option+K`

**Comment by HazAT** [comment]
This has been fixed in `5.0.0-beta.1`
